src/mac/activate-apps-cmake.sh

```diff
@@ -57,13 +57,6 @@ done
 
 echo "$lst"
 
-osascript -e 'tell application "Finder"'\
- -e 'activate'\
- -e 'make new Finder window'\
- -e 'set target of Finder window 1 to folder '"$lst"' of startup disk'\
- -e 'set target of Finder window 1 to folder "bin" of folder '"$lst"' of startup disk'\
- -e 'end tell'
-
```

Comment:
This is a very old part of the installer (don't know if it dates as far back os OS9) and is intended to tickle MACOS into realising a program is to be launched when one double clicks the icon. If MACOS has moved on to where it is no longer needed then great. Someone needs to see if the icons works after a build into /Applications/NEURON and see if they work after a pkg install. I guess that should be me unless you have already done so.  I take it there is no problem with the wheels? (I don't know if this script is run when you build a wheel).

Reply:
Yep. The icons work after build an package install.